test/Bouncer.test.js

```diff
@@ -0,0 +1,67 @@
+
+import assertRevert from './helpers/assertRevert';
+
+const Bouncer = artifacts.require('BouncerMock');
+
+require('chai')
+  .use(require('chai-as-promised'))
+  .should();
+
+const getSigner = (contract, signer) => (addr) => {
+  // via: https://github.com/OpenZeppelin/zeppelin-solidity/pull/812/files
+  const message = contract.address.substr(2) + addr.substr(2);
+  // ^ substr because in solidity the address is a set of byes, not a string `0xabcd`
+  return web3.eth.sign(signer, web3.sha3(message, { encoding: 'hex' }));
+};
+
+contract('Bouncer', ([owner, authorizedUser, anyone, bouncerAddress]) => {
+  let bouncer;
+  let roleBouncer, roleOwner;
+
+  before(async () => {
```

Comment:
Arrow functions are [discouraged](https://mochajs.org/#arrow-functions) in Mocha because `this` is actually valuable. The `this` object is reset for every test, and `before` blocks (etcetera) have access to it, so it's useful to share values between those two "contexts".

In fact, here `bouncer`, `roleBouncer`, `roleOwner` should be members of `this` instead of variables.